src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStartedExtensionPoint.ts

```diff
@@ -103,6 +103,10 @@ export const walkthroughsExtensionPoint = ExtensionsRegistry.registerExtensionPo
 															hc: {
 																description: localize('walkthroughs.steps.media.image.path.hc.string', "Path to the image for hc themes, relative to extension directory."),
 																type: 'string',
+															},
```

Comment:
FYI these images don't seem to be showing up for me

<img width="975" alt="CleanShot 2022-03-10 at 17 23 28@2x" src="https://user-images.githubusercontent.com/25163139/157783644-10bd9cd5-efaa-41b9-b661-6ed6ffdb9eb5.png">